benchmarks/runners/qcdnum_bench.py

```diff
@@ -26,9 +26,14 @@ def benchmark_lo(self):
         self.run([{}], observables.build(**(observables.default_config[0])), ["ToyLH"])
 
     def benchmark_nlo(self):
+
+        fnames =  {"observable_names": ["F2light", "FLlight", "F3light"],}
+        obs =  observables.default_config[1]
+        obs.update(fnames)
```

Comment:
Pay attention to this: `dict` are mutable objects, so if you change one of them and then use again you will keep using the updated one.
I would suggest to add a `.copy()` call:
```py
        obs =  observables.default_config[1].copy()
        obs.update(fnames)
```

Reply:
okay done